docs/Appendix-A.rst

```diff
@@ -18,32 +18,39 @@ For example, 'pup setup -h' will provide help on the setup function.
 
 Usage;
 
-pup <command> [<args>] [options] [--help | -h]
+pup [command] [<args>] [options] [--help | -h]
 
-Cluster POWER-Up has extensive logging capabilities. To enable detailed
-logging of activities, you can set the log level to debug. To enable
-detailed logging to the logs/gen file, add the -f debug option. To enable
-detailed display of log information, add -p debug. For additional log level
-help, enter -h at the end of a pup command. (ie pup setup -h)
+Cluster POWER-Up has extensive logging capabilities. Logging can take place to
+the screen and a log file (power-up/logs/gen) and the logging level can be
```

Comment:
We should probably change the log file name to something like 'pup.log' (in a subsequent task/commit).